packages/common/src/webrtc/Peer.ts

```diff
@@ -123,12 +123,49 @@ export default class Peer {
     }
   }
 
+  private _sdpHasAudio(sdp: string = ''): boolean {
+    return /m=audio/.test(sdp)
+  }
+
+  private _sdpHasVideo(sdp: string = ''): boolean {
+    return /m=video/.test(sdp)
+  }
```

Comment:
nit: if you want you can move these two methods to the helpers file: `packages/common/src/webrtc/helpers.ts` since these are pure utility functions that do not require the class context.

Reply:
OK, thanks, will do.